mountpoint-s3/src/inode.rs

```diff
@@ -1130,7 +1133,11 @@ impl WriteHandle {
 
                 Ok(())
             }
-            _ => Err(InodeError::InodeNotWritable(inode.err())),
+            WriteStatus::Remote => Err(InodeError::InodeNotWritable(
+                inode.err(),
+                "File already uploaded to S3.".to_owned(),
+            )),
+            WriteStatus::LocalUnopened => unreachable!("Unopened local file cannot be written"),
```

Comment:
I don't think we know this for sure, what if there is a bug and write status becomes `LocalUnopened`? We should also return a reasonable error for this case.

Reply:
Let's discuss this offline, also include @passaro in the discussion.